teps/0056-pipelines-in-pipelines.md

```diff
@@ -485,7 +495,156 @@ These are alternatives to the solution for status discussed above:
 
 ## Design
 
-TODO
+In this section, we flesh out the details of `Pipelines` in `Pipelines` in relation to:
+- [Parameters](#parameters)
+- [Results](#results)
+  - [Consuming Results](#consuming-results)
+  - [Producing Results](#producing-results)
+- [Execution Status](#execution-status)
+- [Workspaces](#workspaces)
+- [When Expressions](#when-expressions)
+- [Retries](#retries)
+- [Timeouts](#timeouts)
+- [Matrix](#matrix)
+
+### Parameters
+
+`Pipelines` in `Pipelines` will consume `Parameters` similarly to `Tasks` in `Pipelines`.
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: clone-scan-notify
+spec:
+  params:
+    - name: repo
+      value: $(params.repo)
+  tasks:
+    - name: git-clone
+      params:
+        - name: repo
+          value: $(params.repo)      
+      taskRef:
+        name: git-clone
+    - name: security-scans
+      params:
+        - name: repo
+          value: $(params.repo)
+      pipelineRef:
+        name: security-scans
+    - name: notification
+      taskRef:
+        name: notification
+```
+
+### Results
+
+#### Consuming Results 
+
+`Pipelines` in `Pipelines` will consume `Results`, similarly to `Tasks` in `Pipelines`.
```

Comment:
Same as with parameters, "similarly" doesn't feel like quite the right phrasing.

Reply:
...and I'm not gonna bother flagging this anywhere else, just assume that I'm not sure about "similarly" for `PipelineTask` behaviors that will actually behave the same in all cases. =)